test/extensions/filters/http/cache/simple_http_cache/simple_http_cache_test.cc

```diff
@@ -107,25 +125,25 @@ class SimpleHttpCacheTest : public testing::Test {
 
 // Simple flow of putting in an item, getting it, deleting it.
 TEST_F(SimpleHttpCacheTest, PutGet) {
-  const std::string RequestPath1("Name");
-  LookupContextPtr name_lookup_context = lookup(RequestPath1);
+  const std::string request_path1("Name");
+  LookupContextPtr name_lookup_context = lookup(request_path1);
   EXPECT_EQ(CacheEntryStatus::Unusable, lookup_result_.cache_entry_status_);
 
   Http::TestResponseHeaderMapImpl response_headers{{"date", formatter_.fromTime(current_time_)},
                                                    {"cache-control", "public,max-age=3600"}};
 
-  const std::string Body1("Value");
-  insert(move(name_lookup_context), response_headers, Body1);
-  name_lookup_context = lookup(RequestPath1);
-  EXPECT_TRUE(expectLookupSuccessWithBody(name_lookup_context.get(), Body1));
+  const std::string body1("Value");
+  insert(move(name_lookup_context), response_headers, body1);
+  name_lookup_context = lookup(request_path1);
+  EXPECT_TRUE(expectLookupSuccessWithBodyAndTrailers(name_lookup_context.get(), body1));
 
-  const std::string& RequestPath2("Another Name");
-  LookupContextPtr another_name_lookup_context = lookup(RequestPath2);
+  const std::string& request_path2("Another Name");
```

Comment:
Same as below: This string should be something like "/name" so it provides a valid path (including the leading slash).